### PR TITLE
docs: add core and cli version alignment note

### PR DESCRIPTION
**Pull request by FrederikSchlemmer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The issue tracks the problem of a missing information about the suitable versions of Angular core and cli. Due to no updates of the v2.angular.io docs there should be a note in the current documentation.

This PR adds a note in the chapter [Angular versioning](https://angular.io/guide/releases#angular-versioning). This note can be linked in the Getting Started later.

Issue Number: #30261

## What is the new behavior?
Added a note about the version alignment between Angular core and cli.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Review by gkalpak — COMMENTED** [review]
Thx for adding the nore. I'll let the docs team give the final approval.

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -31,6 +31,11 @@ The version number is incremented based on the level of change included in the r
 
 * **Patch releases** are low risk, bug fix releases. No developer assistance is expected during update.
 
+<div class="alert is-helpful">
+
+Since version 7 the major versions of Angular core and cli will be aligned. In order to use Angular core version X you also need to use cli version X.
```
Comment (kapunahelewong) [inline aio/content/guide/releases.md]: Please consider the following edit (I've edited to some of the writerly standards - tense, commas, tone, etc.):

> **Note:**  As of Angular version 7, the major versions of Angular core and the CLI are aligned. This means that in order to use the CLI as you develop an Angular app, the version of `@angular/core` and the CLI need to be the same.

**Review by kapunahelewong — DISMISSED** [review]
Thank you for this, @FrederikSchlemmer. I have just one comment with a recommendation that helps align it with the writing guidelines we use. I've tried to stay true to your thought.

**Review by alexeagle — APPROVED** [review]

**Comment by brandonroberts** [comment]
@FrederikSchlemmer will you rebase on latest master. There have been fixes made for CI stability since this PR was opened.

File: aio/content/guide/releases.md (hunk 1 above)
Comment (FrederikSchlemmer) [reply aio/content/guide/releases.md]: I have included the proposed changes!

**Comment by FrederikSchlemmer** [comment]
@brandonroberts I have adopted the proposed changes and performed a rebase. Should be ready to merge!

**Review by IgorMinar — APPROVED** [review]
lgtm

**Pull request closed without merging**